docker-compose.yaml

```diff
@@ -86,8 +86,8 @@ services:
       options:
         max-size: "20m"
     build:
-      context: ./
-      dockerfile: Dockerfile.addValidator
+      context: ./scripts
+      dockerfile: add-validator/Dockerfile.addValidator
```

Comment:
Let's put the dockerfile full path to the context

Reply:
We can't add it to context, as we need to access the common folder.